scripts/nlvr/generate_data_from_erm_model.py

```diff
@@ -0,0 +1,77 @@
+#! /usr/bin/env python
+
+# pylint: disable=invalid-name,wrong-import-position
+import sys
+import os
+import json
+import argparse
+
+sys.path.insert(0, os.path.dirname(os.path.dirname(os.path.abspath(os.path.join(__file__, os.pardir)))))
+
+from allennlp.data.dataset_readers import NlvrDatasetReader
+from allennlp.models import NlvrCoverageSemanticParser
+from allennlp.models.archival import load_archive
+from allennlp.semparse.worlds import NlvrWorld
+
+
+def make_data(input_file: str,
+              output_file: str,
+              archived_model_file: str,
+              max_num_decoded_sequences: int) -> None:
+    reader = NlvrDatasetReader(output_agendas=True)
+    model = load_archive(archived_model_file).model
+    if not isinstance(model, NlvrCoverageSemanticParser):
+        model_type = type(model)
+        raise RuntimeError(f"Expected an archived NlvrCoverageSemanticParser, but found {model_type} instead")
+    # Tweaking the decoder trainer to coerce the it to generate a k-best list. Setting k to 100
+    # here, so that we can filter out the inconsistent ones later.
+    # pylint: disable=protected-access
```

Comment:
This should be inline, after the current line 29.  Putting it right here enables it for the rest of the function, which you don't want.

Reply:
Done. Thanks!